packages/manager/modules/bm-server-components/src/technical-details/technical-details.controller.js

```diff
@@ -14,11 +14,10 @@ export default class BmServerComponentsTechnicalDetailsController {
     this.formattedDataDisks = this.formatDisks('data');
     this.formattedCacheDisks = this.formatDisks('cache');
     this.formattedOsDisks = this.formatDisks('os');
-    this.formattedExtensionCard = get(
-      this.technicalDetails,
-      'storage.raid',
-      '-',
-    );
+    this.formattedExtensionCard =
+      this.technicalDetails.storage?.raid !== 'none'
+        ? this.technicalDetails.storage?.raid
```

Comment:
I think the `?` it's` not required
```suggestion
        ? this.technicalDetails.storage.raid
```

Reply:
Previously it was intended to be the default case with '-'. I don't know if storage is still set.